doc/examples/segmentation/plot_mask_slic.py

```diff
@@ -0,0 +1,67 @@
+"""
+======================
+maskSLIC Demonstration
+======================
+
+This example is about comparing the segmentations obtained using the
+plain SLIC method [1]_ and its masked version maskSLIC [2]_.
+
```

Comment:
Could you please add one sentence here explaining the difference between the two implementations, like the fact that there are less border effects in the maskSLIC? We don't expect users to go and read the publication :-), so let's help them as much as possible.